pombola/south_africa/templates/home.html

```diff
@@ -104,10 +104,10 @@ <h2>
     <div class="wrapper">
         <div class="home__news__row">
             <div class="home__news__survey">
-                <h3>#SAAbailout</h3>
-                <img class="home__news__survey__thumb" src="{% static 'images/SAA survey.jpg' %}" />
-                <h4>SAA: Bail-out or not?</h4>
-                <a href="https://www.pa.org.za/blog/saa-business-rescue-haveyoursay" class="button button--low-case">Have your say</a>
+                <h3>#LandExpropriation</h3>
```

Comment:
I'm going to remove this `<h3>` because the same text is already in the image, and it looks a bit odd with both! (See below)

<img width="367" alt="screen shot 2018-05-31 at 10 20 01" src="https://user-images.githubusercontent.com/22996/40773954-4151f22e-64bc-11e8-8662-d82dad8e3fa1.png">

Reply:
Done in 34127c8

<img width="365" alt="screen shot 2018-05-31 at 10 48 08" src="https://user-images.githubusercontent.com/22996/40775506-2986f0c8-64c0-11e8-9cb7-386f06db3add.png">